samples/gcp-pubsub-source/channel.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Channel
+metadata:
+  name: qux-1
```

Comment:
qux is left in?

Reply:
Why not? This file is meant to be deployed as-is, without needing modification first.